fedimint-client/src/secret.rs

```diff
@@ -0,0 +1,23 @@
+use fedimint_core::core::ModuleInstanceId;
+use fedimint_derive_secret::{ChildId, DerivableSecret};
+
+const TYPE_MODULE: ChildId = ChildId(0);
+const TYPE_BACKUP: ChildId = ChildId(1);
+
+pub trait DeriveableSecretClientExt {
+    fn derive_module_secret(&self, module_instance_id: ModuleInstanceId) -> DerivableSecret;
+    fn derive_backup_secret(&self) -> DerivableSecret;
+}
```

Comment:
Probably self-documenting enough

Reply:
I think doc strings would be nice. Especially on the struct itself.